testing/integration/upgrade_standalone_same_commit_test.go

```diff
@@ -74,7 +74,7 @@ func TestStandaloneUpgradeSameCommit(t *testing.T) {
 			upgradetest.WithUnprivileged(unprivilegedAvailable),
 			upgradetest.WithDisableHashCheck(true),
 		)
-		assert.ErrorContainsf(t, err, fmt.Sprintf("agent version is already %s", currentVersion), "upgrade should fail indicating we are already at the same version")
+		assert.ErrorContains(t, err, "failed to find watcher:")
```

Comment:
Why is it expected to have an error "failed to find watcher:" here? Shouldn't we assert that we end up with a "Completed " upgrade and no error reported in the upgradeDetails ?